db/dcrpg/internal/blockstmts.go

```diff
@@ -99,6 +99,15 @@ const (
 
 	SelectBlockChainRowIDByHash = `select block_db_id from block_chain where this_hash = $1;`
 
+	// SelectBlocksTicketsPrice selects the ticket price and difficulty for the 144th block
+	SelectBlocksTicketsPrice = `select sbits, time, difficulty from blocks where height % 144 = 0 order by time;`
```

Comment:
All 144 occurrences need to be replaced by a parameter set from the `chaincfg.Params.StakeDiffWindowSize`, which is different between mainnet/testnet2/simnet

Reply:
ok. Noted